server/service/src/sync/translations/name.rs

```diff
@@ -280,12 +281,18 @@ impl SyncTranslation for NameTranslation {
             national_health_number,
             // See comment in pull translation
             custom_data_string: _,
+            is_active,
         } = NameRowRepository::new(connection)
             .find_one_by_id(&changelog.record_id)?
             .ok_or(anyhow::Error::msg(format!(
                 "Name row ({}) not found",
                 changelog.record_id
             )))?;
+        if !is_active {
+            return Ok(PushTranslateResult::Ignored(
+                "Ignore pushing soft deleted name".to_string(),
+            ));
```

Comment:
It got deleted in mSupply and we soft-deleted it. So I guess it correct to not push it back...